Feature/add pizza fact

paulrwade opened the pull request:
## Description

<!--
Please do not leave this blank
This PR [adds/removes/fixes/replaces] the [feature/bug/etc].
-->

## What type of PR is this? (check all applicable)

- [X ] ☝️ Add a pizza fact or trivia
- [ ] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [ ] 📝 Documentation Update

## Related Tickets & Documents

<!--
Please use this format link issue numbers: Fixes/Relates to #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
-->

## Added to documentation?

- [ ] 📜 README.md
- [X ] 🙅 no documentation needed

## [optional] What GIF best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

adiati98 commented on README.md:
```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <br>
-  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="Open Sauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png" width="300px">
```
Please revert this.

paulrwade replied:
I used “git revert” and it appears that it is reverted back now.  Look good?

adiati98 replied:
There is no new commit from you.

After you did `git revert`, you need to:

- add and commit your changes with `git add` and `git commit`
- push them to your remote branch with `git push`.

paulrwade commented:
Hi, Ayu. Thanks again for all your help.

I did a git revert on the logo-update change to README.

I certainly hope I did it correctly. Can you please double-check?

Thanks for helping me to learn this.

Sent with [Proton Mail](https://proton.me/) secure email.

On Tuesday, May 7th, 2024 at 1:58 PM, Ayu Adiati ***@***.***> wrote:

> Hi ***@***.***(https://github.com/paulrwade),
>
> We want to have one PR for one issue. And you have two issues that you work on in this PR.
>
> That said, can you please revert your commit to change the image on the README? Also because it's been worked on by another contributor.
>
> You can read [this article](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to revert your commit.
>
> If you find it overwhelming, you can close this PR and create a new one.
> But you need to do it from scratch by creating a new branch from the main branch.
>
> Please let me know if you have questions or need any help!
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I6JT5HWNJQLDRX2BS3ZBE6APAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGI4TSOJTHA).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

adiati98 reviewed (changes requested):
Hey @paulrwade,

Based on [this comment](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938), please revert the change for the image.

Thank you.

adiati98 commented:
> Hi, Ayu. Thanks again for all your help. I did a git revert on the logo-update change to README. I certainly hope I did it correctly. Can you please double-check? Thanks for helping me to learn this. Sent with [Proton Mail](https://proton.me/) secure email.
> […](#)
> On Tuesday, May 7th, 2024 at 1:58 PM, Ayu Adiati ***@***.***> wrote: Hi ***@***.***(https://github.com/paulrwade), We want to have one PR for one issue. And you have two issues that you work on in this PR. That said, can you please revert your commit to change the image on the README? Also because it's been worked on by another contributor. You can read [this article](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to revert your commit. If you find it overwhelming, you can close this PR and create a new one. But you need to do it from scratch by creating a new branch from the main branch. Please let me know if you have questions or need any help! — Reply to this email directly, [view it on GitHub]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I6JT5HWNJQLDRX2BS3ZBE6APAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGI4TSOJTHA). You are receiving this because you were mentioned.Message ID: ***@***.***>

I haven't seen it changed, @paulrwade. Have you pushed your changes?

paulrwade commented:
I check the opensauce/pizza-verse and confirmed that my PR is cancelled. I ran a revert on my fork paulrwade/pizza-verse and pushed it to my fork. I then checked that the README file in the opensauce/pizza-verse fork contained the original logo and none of my changes.

Your thoughts?

Sent with [Proton Mail](https://proton.me/) secure email.

On Tuesday, May 7th, 2024 at 2:36 PM, Ayu Adiati ***@***.***> wrote:

>> Hi, Ayu. Thanks again for all your help. I did a git revert on the logo-update change to README. I certainly hope I did it correctly. Can you please double-check? Thanks for helping me to learn this. Sent with [Proton Mail](https://proton.me/) secure email.
>> […](#)
>> On Tuesday, May 7th, 2024 at 1:58 PM, Ayu Adiati @.> wrote: Hi @.(https://github.com/paulrwade), We want to have one PR for one issue. And you have two issues that you work on in this PR. That said, can you please revert your commit to change the image on the README? Also because it's been worked on by another contributor. You can read [this article](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to revert your commit. If you find it overwhelming, you can close this PR and create a new one. But you need to do it from scratch by creating a new branch from the main branch. Please let me know if you have questions or need any help! — Reply to this email directly, [view it on GitHub]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I6JT5HWNJQLDRX2BS3ZBE6APAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGI4TSOJTHA). You are receiving this because you were mentioned.Message ID: @.***>
>
> I haven't seen it changed, ***@***.***(https://github.com/paulrwade). Have you pushed your changes?
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099350562), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I2XN6NQMLP5Y2R446DZBFCOJAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGM2TANJWGI).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

adiati98 commented:
> I check the opensauce/pizza-verse and confirmed that my PR is cancelled. I ran a revert on my fork paulrwade/pizza-verse and pushed it to my fork. I then checked that the README file in the opensauce/pizza-verse fork contained the original logo and none of my changes. Your thoughts? Sent with [Proton Mail](https://proton.me/) secure email.
> […](#)
> On Tuesday, May 7th, 2024 at 2:36 PM, Ayu Adiati ***@***.***> wrote: > Hi, Ayu. Thanks again for all your help. I did a git revert on the logo-update change to README. I certainly hope I did it correctly. Can you please double-check? Thanks for helping me to learn this. Sent with [Proton Mail](https://proton.me/) secure email. > […](#) > On Tuesday, May 7th, 2024 at 1:58 PM, Ayu Adiati @.> wrote: Hi @.(https://github.com/paulrwade), We want to have one PR for one issue. And you have two issues that you work on in this PR. That said, can you please revert your commit to change the image on the README? Also because it's been worked on by another contributor. You can read [this article](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to revert your commit. If you find it overwhelming, you can close this PR and create a new one. But you need to do it from scratch by creating a new branch from the main branch. Please let me know if you have questions or need any help! — Reply to this email directly, [view it on GitHub]([#82 (comment)]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938))), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I6JT5HWNJQLDRX2BS3ZBE6APAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGI4TSOJTHA). You are receiving this because you were mentioned.Message ID: @.***> I haven't seen it changed, ***@***.***(https://github.com/paulrwade). Have you pushed your changes? — Reply to this email directly, [view it on GitHub]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099350562)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I2XN6NQMLP5Y2R446DZBFCOJAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGM2TANJWGI). You are receiving this because you were mentioned.Message ID: ***@***.***>

Can you confirm your current branch?
Are you at `feature/add_pizza_fact` branch? You need to be in this branch to revert your commit.

paulrwade commented:
I will delete the other PR from where I added to the pizza trivia.

Here is my revert. Don't know why you keep seeing the logo change in the code after I reverted it.

[image.png]

On Tuesday, May 7th, 2024 at 2:49 PM, Ayu Adiati ***@***.***> wrote:

> @adiati98 commented on this pull request.
>
> ---------------------------------------------------------------
>
> In [README.md](https://github.com/open-sauced/pizza-verse/pull/82#discussion_r1593144595):
>
>> @@ -1,6 +1,6 @@
>  <div align="center">
>    <br>
> -  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
> +  <img alt="Open Sauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png" width="300px">
>
> There is no new commit from you.
>
> After you did git revert, you need to:
>
> - add and commit your changes with git add and git commit
> - push them to your remote branch with git push.
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/open-sauced/pizza-verse/pull/82#discussion_r1593144595), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3IY6DQL444CF56DZRJTZBFD7HAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBUGI2TOMRYG4).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

paulrwade commented:
I was in the wrong branch. I just went ahead and got rid of both of my PRs.

At this point, the opensauced/pizza-verse repo shouldn't have been affected, correct?

I will be more careful about what branch I'm working on. I see now how my second PR inadvertently ALSO included the edited README file. My bad.

Sent with [Proton Mail](https://proton.me/) secure email.

On Tuesday, May 7th, 2024 at 2:57 PM, Ayu Adiati ***@***.***> wrote:

>> I check the opensauce/pizza-verse and confirmed that my PR is cancelled. I ran a revert on my fork paulrwade/pizza-verse and pushed it to my fork. I then checked that the README file in the opensauce/pizza-verse fork contained the original logo and none of my changes. Your thoughts? Sent with [Proton Mail](https://proton.me/) secure email.
>> […](#)
>> On Tuesday, May 7th, 2024 at 2:36 PM, Ayu Adiati @.> wrote: > Hi, Ayu. Thanks again for all your help. I did a git revert on the logo-update change to README. I certainly hope I did it correctly. Can you please double-check? Thanks for helping me to learn this. Sent with [Proton Mail](https://proton.me/) secure email. > […](#) > On Tuesday, May 7th, 2024 at 1:58 PM, Ayu Adiati @.> wrote: Hi @.(https://github.com/paulrwade), We want to have one PR for one issue. And you have two issues that you work on in this PR. That said, can you please revert your commit to change the image on the README? Also because it's been worked on by another contributor. You can read [this article](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to revert your commit. If you find it overwhelming, you can close this PR and create a new one. But you need to do it from scratch by creating a new branch from the main branch. Please let me know if you have questions or need any help! — Reply to this email directly, [view it on GitHub]([[#82](https://github.com/open-sauced/pizza-verse/pull/82) (comment)]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099299938))), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I6JT5HWNJQLDRX2BS3ZBE6APAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGI4TSOJTHA). You are receiving this because you were mentioned.Message ID: @.> I haven't seen it changed, @.(https://github.com/paulrwade). Have you pushed your changes? — Reply to this email directly, [view it on GitHub]([#82 (comment)](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099350562)), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3I2XN6NQMLP5Y2R446DZBFCOJAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGM2TANJWGI). You are receiving this because you were mentioned.Message ID: @.>
>
> Can you confirm your current branch?
> Are you at feature/add_pizza_fact branch? You need to be in this branch to revert your commit.
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/open-sauced/pizza-verse/pull/82#issuecomment-2099381976), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BDDT3IZC32EJB4CMYTL3RSTZBFE5TAVCNFSM6AAAAABHLW5PZWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJZGM4DCOJXGY).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

adiati98 commented:
@paulrwade because you said that you've pushed the changes to revert your commit, when I see that it doesn't get through here, I'm pretty sure that you are on another branch. That's why I want to confirm if you are on this branch.

I saw now that you've made change directly on the `main` branch in your other closed PR. FYI, you always want to create a new branch **before** you work on your changes.
Changes that are made directly on the `main` branch would not be accepted in any open source project because it can cause problems.

You can read [this blog post](https://dev.to/opensauced/the-secret-recipe-to-getting-your-pull-requests-reviewed-and-merged-faster-1fag) for further information. :)

adiati98 commented:
> I see now how my second PR inadvertently ALSO included the edited README file

This happens because you first made changes **directly** on the `main` branch. Then, after you made the changes on the `main` branch, you create a new branch.

So, your first changes got carried on to this new branch.

This is one of the reasons why we don't want to mess with `main` branch. 😊

**Tips:** When you want to create a new branch, always create it from the default (usually `main`) branch. Don't create a new branch from any of your working branch.
